definitions/npm/ramda_v0.26.x/flow_v0.104.x-/test_ramda_v0.26.x_list.js

```diff
@@ -822,4 +818,26 @@ const str: string = "hello world";
     s: a,
     y: b
   }))([1, 2, 3])(["1", "2", "3"]);
+
+  describe('nth', () => {
+    it('should pass', () => {
```

Comment:
I realize this is easier to write, but it's harder to reason about. Let's break these out into individual tests which assert what you're attempting and what you expect the outcome to be (within the `it` text). Otherwise `describe` and `it` have little value here.

Reply:
Will do.